great_expectations/render/__init__.py

```diff
@@ -125,6 +126,499 @@ class LegacyDescriptiveRendererType(str, Enum):
     )
 
 
+class RenderedContent:
```

Comment:
Can we move this and other class definitions out of `__init__`?
Maybe put them in a `render/core.py` or `rendered_content.py`, and import them into the `__init__.py` namespace to preserve existing imports